resources/views/partials/sidebar.blade.php

```diff
@@ -1,15 +1,12 @@
-<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/solid.js" integrity="sha384-+Ga2s7YBbhOD6nie0DzrZpJes+b2K1xkpKxTFFcx59QmVPaSA8c7pycsNaFwUK6l" crossorigin="anonymous"></script>
-<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/fontawesome.js" integrity="sha384-7ox8Q2yzO/uWircfojVuCQOZl+ZZBg2D2J5nkpLqzH1HY0C1dHlTKIbpRz/LG23c" crossorigin="anonymous"></script>
-
 <div class="wrapper">
     <nav id="sidebar">
         <div id="dismiss">
-            <i class="fas fa-angle-left"></i>
+            <i class="fo-icon icon-left-open">&#xe801;</i>
         </div>
 
         <div class="sidebar-header">
             <h3>
-                {{ Auth::user()->name }}
+                    {{ Auth::user()->name }}
```

Comment:
change to  {{ $user->first_name }} {{ $user->last_name }}
(yeah, I know this is my fault)

Reply:
Apart from that we cannot use `$user`, since this variable is not set, I've changed it.